source/eventcore/drivers/posix/sockets.d

```diff
@@ -45,6 +47,15 @@ version (linux) {
 	}
 	else
 		import core.sys.linux.netinet.in_ : IP_ADD_MEMBERSHIP, IP_MULTICAST_LOOP;
+
+	// Linux-specific TCP options
+	// https://github.com/torvalds/linux/blob/master/include/uapi/linux/tcp.h#L95
+	// Some day we should siply import core.sys.linux.netinet.tcp;
+	enum SOL_TCP = 6;
```

Comment:
Because we had run into similar situations, there should be a `static if (!is(typeof(X)))` before each `enum X = y;`, so that future DMD versions can add the constants without breaking existing code.